### ISSUE TITLE
Could not import pyspedas in google colab

### ISSUE THREAD

**Issue by donglai96** [spedas/pyspedas#26]
I was trying to use pyspedas in google colab, the kernel crashed with warning:
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.

**Comment by DecSnowFlake** [comment]
I'm meeting the same issue, do you know how to you solve this problem?

**Comment by donglai96** [comment]
pytplot is using Qt which can not run on the remote server, I guess you could use the source code download only or not plot

**Comment by bryan-harter** [comment]
Just FYI, the latest version of PyTplot should work a little bit better in google colab.  

There have always been try/except blocks that stop importing the Qt graphics if any of the imports fail, which then turns on "no graphics" mode for PyTplot.  However, in google colab I see that all Qt functions are imported properly, but then the entire kernel crashes when attempting to start a Qt app.  So I moved some more functions in the try/except block in an effort to catch it, and I also added a check at the very beginning of the __init__.py file that turns off graphics if it detects PyTplot is running in specifically in google colab.
